utils/removeMarkdown.ts

```diff
@@ -0,0 +1,29 @@
+export default function removeMarkdown(md: string) {
+  return (
+    md
+      .replace(/<[^>]*>/g, '')
+      .replace(/^[=\-]{2,}\s*$/g, '')
+      .replace(/\[(.*?)\][\[\(].*?[\]\)]/g, '$1')
+
+      .replace(/^\s{0,3}>\s?/g, '')
+      .replace(/^\s{1,2}\[(.*?)\]: (\S+)( ".*?")?\s*$/g, '')
+      // Remove atx-style headers
+      .replace(/^(\n)?\s{0,}#{1,6}\s+| {0,}(\n)?\s{0,}#{0,} {0,}(\n)?\s{0,}$/gm, '$1$2$3')
+      // Remove emphasis (repeat the line to remove double emphasis)
+      .replaceAll('**', '')
```

Comment:
I don't think we can use replaceAll yet. Vercel uses Node 14 and [this was added in v15](https://nodejs.org/en/blog/release/v15.0.0/#v8-8-6-35415).